sdk/core/core/src/az_http_policy_logging.c

```diff
@@ -158,29 +158,29 @@ void _az_http_policy_logging_log_http_response(
 }
 
 AZ_NODISCARD az_result az_http_pipeline_policy_logging(
-    _az_http_policy* policies,
-    void* options,
-    _az_http_request* ref_request,
-    az_http_response* ref_response)
+    _az_http_policy* p_policies,
+    void* p_data,
+    _az_http_request* p_request,
+    az_http_response* p_response)
 {
-  (void)options;
+  (void)p_data;
 
   if (az_log_should_write(AZ_LOG_HTTP_REQUEST))
   {
-    _az_http_policy_logging_log_http_request(ref_request);
+    _az_http_policy_logging_log_http_request(p_request);
   }
 
   if (!az_log_should_write(AZ_LOG_HTTP_RESPONSE))
   {
     // If no logging is needed, do not even measure the response time.
-    return az_http_pipeline_nextpolicy(policies, ref_request, ref_response);
```

Comment:
`policies, ref_request, ref_response`is the ideal naming. We don't use the `p_` convention anomore, we should change it to the other way.
```
somestruct const* options // means it is read-only and we will not modify it
somestruct* ref_somestruct //means we read it and (may) modify it
somestruct* out_somestruct // means we don't read it, the customer can provide an uninitialized struct, and we will fill it in as a returning result
``

Reply:
Yes, I like these conventions - please apply them